app/views/unattended/report_templates/subscription_-_entitlement_report.erb

```diff
@@ -39,8 +39,8 @@ require:
           'Products': host_products_names(host),
           'Subscription Name': sub_name(pool),
           'Subscription Type': pool.type,
-          'Subscription Quantity': pool.quantity,
-          'Subscriptions Consumed': pool.consumed,
+          'Subscription Total Quantity': pool.quantity,
+          'Subscriptions Consumed': host.filtered_entitlement_quantity_consumed(pool),
```

Comment:
Should we rename this column to make it clear that it's specific to the host? Users who already use this report may get confused when the number changes..

Reply:
I updated the names.